docs/supported-languages.md

```diff
@@ -30,127 +30,47 @@ Object.entries(frontMatter).filter(
 
 This document provides information about supported languages and language maturity definitions for the following products:
 
-* Semgrep OSS Engine
+* Semgrep Code
 * Semgrep Supply Chain
-* Semgrep Pro Engine
 
-## Semgrep OSS Engine
+## Semgrep Code 
+
+Secure your code quickly and continuously by scanning with Semgrep Code, a fast and lightweight SAST (Static Application Security Testing) product that leverages Semgrep OSS Engine and Semgrep Pro Engine. Semgrep Code helps you quickly find and fix vulnerabiliies in your code base. 
```

Comment:
Maybe add 1 line explaining Semgrep OSS Engine and Semgrep Pro Engine, then add a link?